ci/ray_ci/doc_builder_container.py

```diff
@@ -4,11 +4,13 @@
 class DocBuilderContainer(Container):
     def __init__(self) -> None:
         super().__init__("docbuild")
+        self.install_ray(build_type="doc")
 
     def run(self) -> None:
         self.run_script(
             [
                 "cd doc",
+                "pip install -r requirements-doc.txt",
```

Comment:
you probably don't need this, since this is already been installed in doc.build.Dockerfile (the docbuild image in line 6)